app/javascript/article-form/components/Help/index.jsx

```diff
@@ -0,0 +1,94 @@
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { ArticleFormTitle } from './ArticleFormTitle';
+import { TagInput } from './TagInput';
+import { BasicEditor } from './BasicEditor';
+import { EditorFormattingHelp } from './EditorFormattingHelp';
+import { Modal } from '@crayons';
+
+const renderModal = (onClose, title, selector) => {
+  const helpHtml =
+    document.getElementById(selector) &&
+    document.getElementById(selector).innerHTML;
```

Comment:
suggestion (not-blocking) - I think you can use optional chaining to get the same effect here

```suggestion
    document.getElementById(selector)?.innerHTML;
```